ui/lib/apps/Statement/components/StatementsOverview.tsx

```diff
@@ -196,16 +201,20 @@ export default function StatementsOverview({
   const [columns, setColumns] = useState<IColumn[]>([])
   const [visibleColumnKeys, setVisibleColumnKeys] = useState<{
     [key: string]: boolean
-  }>({
-    digest_text: true,
-    sum_latency: true,
-    avg_latency: true,
-    exec_count: true,
-    avg_mem: true,
-    related_schemas: true,
-  })
+  }>(() =>
+    load(VISIBLE_COLUMN_KEYS, {
```

Comment:
you can use https://hooks.umijs.org/state/use-local-storage-state

Reply:
cool! will try it soon!

Reply:
nice hook!